frontend/apps/web/package.json

```diff
@@ -52,6 +52,7 @@
     "cron-validate": "^1.4.5",
     "date-fns": "^3.6.0",
     "date-fns-tz": "^3.1.3",
+    "moment": "^2.30.1",
```

Comment:
Do we need this? It's a been dependency and if we can we should prefer to stick to `date-fns` instead.

Reply:
not seeing it used anywhere - removing it